tests/commands.spec.ts

```diff
@@ -779,6 +779,46 @@ describe("commands unit tests", () => {
         });
     });
 
+    describe("getServiceInstances", () => {
+
+        it("on success", async () => {
+            vscodeWindowMock.expects("withProgress").withArgs({ location: nsVsMock.testVscode.ProgressLocation.Notification, title: messages.loading_services, cancellable: true }).resolves([{}]);
+            const result = await commands.getServiceInstances();
+            expect(result).to.have.lengthOf(1);
+        });
+
+        it("on any error", async () => {
+            const errorMessage = "test error";
+            vscodeWindowMock.expects("withProgress").withArgs({ location: nsVsMock.testVscode.ProgressLocation.Notification, title: messages.loading_services, cancellable: true }).rejects(new Error(errorMessage));
+            try {
+                await commands.getServiceInstances();
+                expect(true).to.be.equal(false);
```

Comment:
```suggestion
                fail("test should fail here");
```

Reply:
fix, note that there is a better way to handle exception but I didn't want to change all the test 
read about "chai-as-promised" package